content/courses/program-optimization/lookup-tables.md

```diff
@@ -77,18 +93,27 @@ the following parameters:
 You then transform this message object into a version `0` transaction using the
 `compileToV0Message()` method.
 
-```typescript
+Replace the `<YOUR_QUICKNODE_URL_HERE>` placeholder with the URL of your RPC.
+You can get a RPC providers from [Solana RPC](https://solana.com/rpc).
```

Comment:
We shouldn't advertise one particular RPC provider. This is mentioned in CONTRIBUTING.md